common/perf-test-core/src/main/java/com/azure/perf/test/core/ExceptionTest.java

```
@@ -26,4 +28,19 @@ public void run() {
     public Mono<Void> runAsync() {
         return Mono.error(new IllegalArgumentException());
     }
+
+    @Override
+    public CompletableFuture<Void> runAsyncWithCompletableFuture() {
+        return null;
```

Should these new methods return errors instead of `null`?